basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
-- Supports IP allowlist
-- Supports audit
-
-## Tools
-
-- Supports fast backup
-- Supports data migration from MySQL to TiDB using tools
-- Supports deploying and maintaining TiDB using tools
+# TiDB Features
+
+The following table provides an overview of the feature development history of TiDB. Note that features under active development may change before final release.
+
+| Data types, functions and operators                                                  | Development | 4.0 | 3.1 | 3.0 | 2.1 |
+|--------------------------------------------------------------------------------------|:-----:|:---:|:---:|:---:|:---:|
+| [Numeric types](/data-type-numeric.md)                                               | Y     | Y   | Y   | Y   | Y   |
+| [Date and time types](/data-type-date-and-time.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| [String types](/data-type-string.md)                                                 | Y     | Y   | Y   | Y   | Y   |
+| [JSON type](/data-type-json.md)                                                      | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Control flow functions](/functions-and-operators/control-flow-functions.md)         | Y     | Y   | Y   | Y   | Y   |
+| [String functions](/functions-and-operators/string-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Numeric functions and operators](/functions-and-operators/numeric-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Date and time functions](/functions-and-operators/date-and-time-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Bit functions and operators](/functions-and-operators/bit-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Cast functions and operators](/functions-and-operators/cast-functions-and-operators.md) | Y     | Y   | Y   | Y   | Y   |
+| [Encryption and compression functions](/functions-and-operators/encryption-and-compression-functions.md) | Y     | Y   | Y   | Y   | Y   |
+| [Information functions](/functions-and-operators/information-functions.md)           | Y     | Y   | Y   | Y   | Y   |
+| [JSON functions](/functions-and-operators/json-functions.md)                         | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Aggregation functions](/functions-and-operators/aggregate-group-by-functions.md)    | Y     | Y   | Y   | Y   | Y   |
+| [Window functions](/functions-and-operators/window-functions.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Miscellaneous functions](/functions-and-operators/miscellaneous-functions.md)       | Y     | Y   | Y   | Y   | Y   |
+| [Operators](/functions-and-operators/operators.md)                                   | Y     | Y   | Y   | Y   | Y   |
+| **Character sets**                                                                   | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
```

Comment:
I think it's a bit strange to link all these character sets to the same doc file. What about removing the link below and adding one here?
```suggestion
| [**Character sets**](/character-set-and-collation.md)                                                                   | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
```